content/threads_intro.tex

```diff
@@ -0,0 +1,63 @@
+This section specifies the interaction between \openshmem interfaces and the
+user threads, and also describes the routines that can be used for initializing and
+querying the thread environment. There are four levels of threading supported by
+the \openshmem implementation.
+
+\begin{itemize}
+\item {\bf \CONST{SHMEM\_THREAD\_SINGLE}} The \openshmem program may not be multithreaded.
```

Comment:
Styling: Needs more separation between item and its description.

Reply:
Also, [`\bf` should be `\textbf`](https://tex.stackexchange.com/a/41718) unless we want `\bf` to stomp all over whatever the actual styling of `\CONST` is. This suggestion is a general issue with the document though.

Reply:
Captured in #121